<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StepByStepForm</class>
 <widget class="QWidget" name="StepByStepForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableBuffer">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>660</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableProcessor">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>0</y>
     <width>391</width>
     <height>660</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableSystem">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>403</width>
     <height>660</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="nextStepButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>660</y>
     <width>1200</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Next Step</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
